src/routes/pridegui/+page.svelte.md

```diff
@@ -1,44 +1,48 @@
 <!-- ^ what does this meannnn -->
-# PrideGUI
 
-![Modrinth Downloads](https://img.shields.io/modrinth/dt/6mcKx2Pb?logo=modrinth&label=Modrinth%20downloads&color=%231bd96a&style=for-the-badge) ![GitHub Downloads (all assets, all releases)](https://img.shields.io/github/downloads/pridecraft-studios/pridegui/total?style=for-the-badge&logo=github&label=Github%20Downloads) ![GitHub commit activity](https://img.shields.io/github/commit-activity/t/pridecraft-studios/pridegui?style=for-the-badge&logo=github) 
+# PrideGUI
 
-This, is PrideGUI. An official addon to [PridePack](https://modrinth.com/resourcepack/pridepack) that changes the Minecraft GUI to be LGBT Themed. You can use it as a standalone pack or an addon to Pridepack.
-It comes in a single, Dark version.
+<aside class="shields">
+<Modrinth modid="pridegui"/>
+<GitHubDownloads modid="pridegui" />
+<GitHubCommits modid="pridegui" />
+</aside>
 
-It's first release is 18.0.0, due to the Pridepack Versioning System.
+This is an official addon to [PridePack](/pridepack) that changes the Minecraft GUI to have a dark theme,
+with LGBT details.
+It is available for many versions, starting from Beta 1.7.3.
+You can get [PrideGUI Legacy here](https://modrinth.com/project/pridegui-legacy),
+which has versions of the pack for Beta 1.7.3 all the way through 1.20.1.
 
 ![PrideGUI banner](https://cdn.modrinth.com/data/6mcKx2Pb/images/05250c177741152dca8e964ea47a3806d23d4432.png)
-## Who?
-
-We are [Pridecraft Studios](https://pridecraft.gay), A team that makes LGBT content for Minecraft. We're working on [PridePack](https://git.pridecraft.gay/PridePack), [PrideMod](https://git.pridecraft.gay/PrideMod), and a few other [projects](https://git.pridecraft.gay/repositories).
-
-## How can I test new stuff out?
-
-Currently, the best way to test stuff out is to join [our Discord](https://discord.pridecraft.gay) or follow [our Mastodon](https://tech.lgbt/@pridecraft) or [our BlueSky](https://bsky.app/profile/pridecraft.gay) to see sneak-peaks of new features. 
 
-## How can I contribute?
-You can help us translate Pridepack to more languages on [Crowdin](https://translate.pridecraft.gay) [![Crowdin](https://badges.crowdin.net/pridepack/localized.svg)](https://crowdin.com/project/pridepack).
-You can also indirectly contribute by voting on polls on [the Discord](https://discord.pridecraft.gay), or [our Mastodon](https://tech.lgbt/@pridecraft)
+<Donate/>
 
-## What mods are currently supported?
-Currently, there is no support planned for PrideGUI just yet, but mod support may be added in the future, after the initial project is finished.
+<Who/>
 
+## Where can I download PrideGUI?
 
-## Where is this avaliable?
+<ul class="badges">
+<li><Badge id="modrinth" rel="me" link="https://modrinth.com/resourcepack/pridegui" ext="svg" head="Available on" name="Modrinth"/></li>
+<li><Badge id="github" rel="me" link="https://github.com/Pridecraft-Studios/pridegui" ext="svg" head="Available on" name="GitHub"/></li>
+</ul>
 
```

Comment:
![image](https://github.com/user-attachments/assets/b4b62906-f7d6-4874-96d7-480046942b87)
PrideGUI should have download links to all different editions (Normal, Legacy, Light Mode, Light Mode Legacy).

You can use less space by using the smaller badge design, like we do on Modrinth. On the website it could be separated in 4 collumns if you can figure it out

![image](https://github.com/user-attachments/assets/07bad413-ac65-4f18-9df3-316c0a14eb93)

Reply:
It already shrinks for smaller views automatically; although I'd probably going to want to make that explicit; hm.

I'll figure something out; it'll probably be 4 columns that collapse to fit to the screen.

Reply:
I meant you could use the smaller ones by default (on non W    I    D     E screens) to take up less space. Quite neat that it shrinks automatically.

Speaking of smaller screens, I should test it on mobile

Reply:
![Screenshot_20241016-231335](https://github.com/user-attachments/assets/72d6f85e-6a29-4648-a612-673b8ebc1673)
Wasn't it meant to shrink?

Reply:
Hm, possible your display is considered too wide for the CSS to trip on that.